x-pack/plugins/security_solution/public/timelines/components/timeline/body/helpers.test.ts

```diff
@@ -242,4 +248,60 @@ describe('helpers', () => {
       expect(eventIsPinned({ eventId, pinnedEventIds })).toEqual(false);
     });
   });
+
+  describe('isInvestigateInResolverActionEnabled', () => {
+    it('returns false if agent.type does not equal endpoint', () => {
+      const data: Ecs = { _id: '1', agent: { type: ['blah'] } };
+
+      expect(isInvestigateInResolverActionEnabled(data)).toBeFalsy();
+    });
+
+    it('returns false if agent.type does not have endpoint in first array index', () => {
+      const data: Ecs = { _id: '1', agent: { type: ['blah', 'endpoint'] } };
+
+      expect(isInvestigateInResolverActionEnabled(data)).toBeFalsy();
+    });
+
+    it('returns false if process.entity_id is not defined', () => {
+      const data: Ecs = { _id: '1', agent: { type: ['endpoint'] } };
+
+      expect(isInvestigateInResolverActionEnabled(data)).toBeFalsy();
+    });
+
+    it('returns false if process.entity_id is an empty array', () => {
+      const data: Ecs = { _id: '1', agent: { type: ['endpoint'] } };
```

Comment:
fyi, this is missing the empty array

Reply:
Ah thanks. I actually check that case below. Must be an accidental copy paste. I'll remove it.